lib/open_project/backlogs/hooks.rb

```diff
@@ -80,40 +80,43 @@ def view_work_packages_form_details_bottom(context = {})
-                $('#issue_start_date').parent().hide();
-                $('#issue_due_date').parent().hide();
-              });
-            }(jQuery))
-          JS
-        end
-      end
-
-      snippet << '<div class="attribute_wrapper">'
-      snippet << context[:form].text_field(:remaining_hours, :size => 3, :class => 'short', :placeholder => l(:label_hours))
-      snippet << '</div>'
-
       params = context[:controller].params
       if work_package.is_story? && params[:copy_from]
-        snippet << "<p><label for='link_to_original'>#{l(:rb_label_link_to_original)}</label>"
-        snippet << "#{check_box_tag('link_to_original', params[:copy_from], true)}</p>"
-
-        snippet << "<p><label>#{l(:rb_label_copy_tasks)}</label>"
-        snippet << "#{radio_button_tag('copy_tasks', 'open:' + params[:copy_from], true, { id: 'copy_tasks_open'})} #{l(:rb_label_copy_tasks_open)}<br />"
-        snippet << "#{radio_button_tag('copy_tasks', 'none', false, {id: 'copy_tasks_none'})} #{l(:rb_label_copy_tasks_none)}<br />"
-        snippet << "#{radio_button_tag('copy_tasks', 'all:' + params[:copy_from], false, {id: 'copy_tasks_all'})} #{l(:rb_label_copy_tasks_all)}</p>"
+        snippet << "<div class=\"form--field\">"
+        snippet << "<label class=\"form--label\"
+                           for='link_to_original'>#{l(:rb_label_link_to_original)}</label>"
+        snippet << "<span class=\"form--field-container\">"
+        snippet << "<span class=\"form--check-box-container\">"
+        snippet << check_box_tag('link_to_original', params[:copy_from], true, class: 'form--check-box')
+        snippet << "</span></span></div>"
+
+        snippet << "<div class=\"form--field\">"
+        snippet << "<label class=\"form--label\">#{l(:rb_label_copy_tasks)}</label>"
+        snippet << "<span class=\"form--field-container -vertical\">"
+        snippet << "<span class=\"form--radio-button-container\">"
+        snippet << "#{radio_button_tag('copy_tasks',
+                                       'open:' + params[:copy_from],
+                                       true,
+                                       { id: 'copy_tasks_open' })}
+                    #{l(:rb_label_copy_tasks_open)}"
+        snippet << "</span>"
+        snippet << "<span class=\"form--radio-button-container\">"
+        snippet << "#{radio_button_tag('copy_tasks',
+                                       'none',
+                                       false,
+                                       { id: 'copy_tasks_none' })}
+                    #{l(:rb_label_copy_tasks_none)}"
+        snippet << "</span>"
+        snippet << "<span class=\"form--radio-button-container\">"
+        snippet << "#{radio_button_tag('copy_tasks',
+                                       'all:' + params[:copy_from],
+                                       false,
+                                       { id: 'copy_tasks_all' })}
```

Comment:
Without being too picky here (and this is probably out of scope anyway) shouldn't we switch to `content_tag` and the rest of the form helpers here? This is not to my taste at all - and very unmaintainable.